legend-engine-ide-lsp-default-extensions/src/main/java/org/finos/legend/engine/ide/lsp/extension/core/PureLSPGrammarExtension.java

```diff
@@ -316,10 +316,10 @@ private Stream<Optional<LegendReferenceResolver>> toReferences(Constraint constr
     @Override
     public Iterable<? extends LegendExecutionResult> execute(SectionState section, String entityPath, String commandId, Map<String, String> executableArgs, Map<String, Object> inputParameters)
     {
-        return FunctionExecutionSupport.EXECUTE_COMMAND_ID.equals(commandId) ?
-                FunctionExecutionSupport.executeFunction(this, section, entityPath, inputParameters)
-                :
-                super.execute(section, entityPath, commandId, executableArgs, inputParameters);
+        return FunctionExecutionSupport.EXECUTE_COMMAND_ID.equals(commandId)
```

Comment:
Wont the execute command also be handled by the function support?  I would expect this to just be

```return FunctionExecutionSupport.execute(this, section, entityPath, commandId, executableArgs, inputParameters);```

Reply:
Done